frontends/web/src/routes/settings/electrum.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2018 Shift Devices AG
```

Comment:
```suggestion
 * Copyright 2022 Shift Devices AG
```

Perhaps you forgot this one :)

Reply:
Added a new copyright line